e2e/cypress/integration/interactive_menu/basic_options.js

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/**
+* Note: This test requires webhook server running. Initiate `npm run start:webhook` to start.
```

Comment:
There's a module that we can use to [start a server](https://docs.cypress.io/guides/guides/continuous-integration.html#Boot-your-server) before doing test but I initially thought that it's not necessary since not all specs/files are using that webhook server.  So I just put a note only as a reminder for local development.  On test server, this will be started via CLI command (PR to follow in shared pipelines).